spec/paperclip/attachment_spec.rb

```diff
@@ -629,7 +740,7 @@ class Paperclip::Test < Paperclip::Processor; end
       @file = StringIO.new("...")
       Paperclip::Test.stubs(:make).returns(@file)
 
-      rebuild_model styles: { normal: '' }, processors: lambda { |a| [ :test ] }
+      rebuild_model styles: { normal: "" }, processors: lambda { |a| [ :test ] }
```

Comment:
Unused block argument - `a`. If it's necessary, use `_` or `_a` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Space inside square brackets detected.